frontend/i18next.scanner/en/translation.json

```diff
@@ -578,5 +575,11 @@
   "{{author}} updated {{task}} on {{content}}{{workspaceInfo}}": "{{author}} updated {{task}} on {{content}}{{workspaceInfo}}",
   "{{author}} deleted {{task}} on {{content}}{{workspaceInfo}}": "{{author}} deleted {{task}} on {{content}}{{workspaceInfo}}",
   "There is no tasks that match your filter": "There is no tasks that match your filter",
-  "You don't have any assigned tasks": "You don't have any assigned tasks"
+  "You don't have any assigned tasks": "You don't have any assigned tasks",
+  "Confidential space": "Confidential space",
+  "Hide space list": "Hide space list",
+  "Show space list": "Show space list",
+  "Space management": "Space management",
+  "Hide my items": "Hide my items",
+  "Show my items": "Show my items"
```

Comment:
I'm just concerned about these that are used in the tooltip, I don't think they are items from the user point of view, maybe something more about a menu user?

Reply:
done